external/iotivity/iotivity_1.2-rel/resource/csdk/stack/include/oickeepalive.h

```diff
@@ -74,7 +74,7 @@ OCStackResult OCFindKeepAliveResource(OCDoHandle *handle, const char *remoteAddr
  * @return ::OC_STACK_OK on success, some other value upon failure.
  */
 OCStackResult OCSendKeepAliveRequest(OCDoHandle *handle, const char *remoteAddr,
-                                     OCPayload *payload, OCCallbackData *cbData);
+                                     OCRepPayload *payload, OCCallbackData *cbData);
```

Comment:
Instead of making changes in the prototype, typecast the parameter in the function call with (OCPayload *).

Reply:
ok